### ISSUE TITLE
Support PHP script parameter values containing spaces

### ISSUE THREAD

**Issue by nachtkinder** [rgbkrk/atom-script#600]
# Summary
When attempting to supply a parameter value containing a space to a PHP script it is necessary to enclose the parameter value in double-quotes.  Doing this in the "Program Arguments" field of Script's "Configure Run Options" dialogue results in the parameter being incorrectly parsed.

# To reproduce

## Create and verify test case
Create space_test.php containing the following:
```php
<?php
$argsCount = $argc -1;
echo "$argsCount argument(s) received";

if ('foo bar' != $argv[1]) {
	trigger_error('Space mishandled', E_USER_ERROR);
}
```

Verify that the script works correctly on the command-line:
```bash
php -f space_test.php -- "foo bar"
```

The result should be:
>1 argument(s) received

## Run with Script package in Atom
Open space_test.php in Atom.

Open Script's "Configure Run Options" dialogue and enter `php` in the "Command" field and `"foo bar"` in "Program Arguments".

Click the "Run" button.

The expected result is as before:
>1 argument(s) received

The actual result is:
>PHP Fatal error:  Space mishandled in space_test.php on line 8
PHP Stack trace:
PHP   1. {main}() space_test.php:0
PHP   2. trigger_error() space_test.php:8
2 argument(s) received

The above was discovered in Atom 1.0.19 and Script 2.29.0 running on Ubuntu 14.04 LTS 64Bit.  `go version` returns:
>go version go1.4.2 linux/amd64

**Comment by blokeley** [comment]
This is causing me issues running a Python script on Windows too. I'm trying to pass a filename as a command line argument. The filename has spaces in it, and Script passes it as 2 arguments:

```python
import sys

for arg in sys.argv:
    print(arg)
```
Gives:

| Command line argument | Output |
|-----------------------------------|----------|
| arg1 arg2 | arg1 <br /> arg2 |
| "arg1 arg2" | "arg1 <br />arg2" |
| arg1\ arg2 | arg1" arg2 |

**Comment by rgbkrk** [comment]
Yeah I knew we'd run into this problem because sometimes people want multiple arguments and there's no quote parsing here currently. Thanks for reporting the bug so thoroughly.

**Comment by Stormherz** [comment]
Working on this
